example/monitoring-stats.txt

```diff
@@ -6,10 +6,6 @@ io.cmd
 io.cmd.{<any elliptics command type in upper case>}
     time of commands for each type
 
-io.cmd.{<any elliptics command type in upper case>}.lock_time
-    time spent in id lock waiting, for each command type,
-    (this will be 0 for commands with DNET_FLAGS_NOLOCK flag set)
-
```

Comment:
Why do you remove this? Isn't it still possible to wait on locked command if there are only already locked keys in the queue?